### DISCUSSION TITLE
Help me to get galaxy brain?

### DISCUSSION THREAD

**Discussion by Esantomi** [Schweinepriester/github-profile-achievements#148] [Q&A] [answered]
If anyone leaves an answer, I will mark it as answer.
We can do it to each other like that.

**Comment by BitTheCat** [comment, marked as answer]
answer, now i create one for you (select this answer ty ^^)

Answer at least 2 discussions.

**Reply by Esantomi** [reply]
Many thanks to you!

**Reply by BitTheCat** [reply]
for u #149
